content/docs/installation/kubectl.md

```diff
@@ -50,17 +50,26 @@ $ kubectl create clusterrolebinding cluster-admin-binding \
 > and potentially broken states. Please ensure you follow the below steps when
 > uninstalling to prevent this happening.
 
-Before continuing, ensure that all cert-manager resources that have been created
+Before continuing, ensure that unwanted cert-manager resources that have been created
 by users have been deleted. You can check for any existing resources with the
 following command:
 
 ```bash
 $ kubectl get Issuers,ClusterIssuers,Certificates,CertificateRequests,Orders,Challenges --all-namespaces
 ```
+It is recommended that you delete all these resources before uninstalling cert-manager. 
+If plan on reinstalling later and don't want to lose some custom resources, you can keep them. 
```

Comment:
You cannot keep the custom resources if you delete the `CustomResourceDefinition`s as per the following steps in these instructions. Users may choose to manage `CustomResourceDefinition`s separately in which case they don't need to delete custom resources - it might be worth to add a note about that somewhere in the doc.